vendors/espressif/boards/esp32/aws_demos/config_files/aws_demo_config.h

```diff
@@ -62,4 +62,21 @@
 
 #define democonfigGREENGRASS_DISCOVERY_TASK_STACK_SIZE    ( configMINIMAL_STACK_SIZE * 12 )
 
+#define democonfigMEMORY_ANALYSIS
+
+#ifdef democonfigMEMORY_ANALYSIS
+    /* configTOTAL_HEAP_SIZE abstracted away to prevent compiler error." */
+    /* ESP32 heap size spans multiple memory locations. Please see ESP32 Heap Documentation to find total available heap size. */
+    #define democonfigMEMORY_ANALYSIS_HEAP_SIZE           0
+    #define democonfigSTACK_DATA_TYPE                     "bytes"
```

Comment:
Is "democondfigSTACK_SIZE_UNITS" a better naming here?

Reply:
Agreed. Will change.